linux_os/guide/system/software/integrity/software-integrity/rpm_verification/rpm_verify_hashes/ansible/shared.yml

```diff
@@ -15,7 +15,6 @@
 
 - name: "Read files with incorrect hash"
   shell: |
-    set -o pipefail
```

Comment:
Can you add a comment why we cannot have `set -o pipefail`?
If possible, can you add comment to suppress warnings from Ansible lint?
Motivation for pipefail is explained in #4230

Reply:
@yuumasato Added comment about `set -o pipefail`, but what do you mean with suppress warning from Ansible lint? Ansible doesn't show any warning to me (tried also without `warn: False` line)

Reply:
@mildas `ansible-lint` command, provided by `python3-ansible-lint`

With your changes I get:
```bash
$ ansible-lint rhel8/playbooks/pci-dss/rpm_verify_hashes.yml 
[306] Shells that use pipes should set the pipefail option
rhel8/playbooks/pci-dss/rpm_verify_hashes.yml:36
Task/Handler: Read files with incorrect hash
```

To suppress this:
https://docs.ansible.com/ansible-lint/rules/rules.html#false-positives-skipping-rules

Reply:
@yuumasato Well, I tried everything and I could suppress this error with `# noqa 306` comment on `name:` line or after `args:` line, but problem with it is that build deletes these comments - final playbook doesn't contain the suppress comment :-(

Reply:
@mildas LOL, OK then.
Thanks for checking.
I'm merging, hopefully the comment will be enough to explain why there is not pipefail.